examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,642 @@
+{
```

Comment:
<p>What about </p><p><code>import numpy as np</code></p><p> </p><p>and then tweak accordingly ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>